examples/chip-tool/commands/common/CommandInvoker.h

```diff
@@ -115,9 +115,11 @@ class CommandInvoker final : public ResponseReceiver<typename RequestType::Respo
         {
             return CHIP_ERROR_NO_MEMORY;
         }
-        ReturnErrorOnFailure(commandSender->SendCommandRequest(session));
+        ReturnErrorOnFailure(commandSender->SendCommandRequest(session.Value()));
 
         commandSender.release();
+        exchangeManager->GetSessionManager()->RemoveGroupSession(session.Value()->AsGroupSession());
```

Comment:
I assume this is a temporary hack and that there is an issue tracking doing this right?  Maybe ~SessionHandle should be doing this, for group sessions?

Reply:
This is no resilient to group key hash collisions I think.

Reply:
I created an issue for it and yes it a temporary fix.
https://github.com/project-chip/connectedhomeip/issues/13507